repo/fsrepo/fsrepo.go

```diff
@@ -578,6 +581,26 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 	return du, err
 }
 
+func (r *FSRepo) SwarmKey() ([]byte, error) {
```

Comment:
Shall the swarm key be part of the ipfs configuration file (as the host identity is?). My first impression is that it would be a more consistent approach.